aries-site/src/examples/templates/Card/Card.js

```diff
@@ -45,14 +61,18 @@ const DesignSystemCard = ({
           >
             {avatar && (
               <Avatar
-                 flex={false}
+                flex={false}
                 pad={{ bottom: 'small' }}
                 a11yTitle={`${title} icon`}
-                round='medium'
+                round="medium"
                 src={avatar}
               />
             )}
-            {icon && <Box pad={{ bottom: 'small' }}>{icon}</Box>}
+            {icon && skeleton ? (
+              <IconSkeleton />
+            ) : (
+              <Box pad={{ bottom: 'small' }}>{icon}</Box>
+            )}
             {pretitle && <Text size="small">{pretitle}</Text>}
             {title && typeof title === 'string' ? (
```

Comment:
in the loading situation wouldn't the title be undefined? We still want the heading there in this situation so it can be a skeleton.

Reply:
Im not sure Im following,.. the `title` is still present in the skeleton render

Reply:
My point is I think it's probably working now because a title is always passed but in a real world situation you wouldn't have the title yet during loading but you still want a `<Heading>` component to provide the structure for the skeletonized view. Ie. this should still render a `<Heading>` with an empty title in a real world example

Reply:
aw okay thank you sorry I wasnt sure what you meant